gwpy/signal/qtransform.py

```diff
@@ -149,6 +157,46 @@ def __iter__(self):
             yield QPlane(q, self.frange, self.duration, self.sampling,
                          mismatch=self.mismatch)
 
+    def transform(self, fseries, **kwargs):
+        """Compute the time-frequency plane at fixed Q with the most
+        significant tile
+
+        Parameters
+        ----------
+        fseries : `~gwpy.timeseries.FrequencySeries`
+            the complex FFT of a time-series data set
+
+        **kwargs
+            other keyword arguments to pass to `QPlane.transform`
+
+        Returns
+        -------
+        result : `QGram`
+            signal energies over the time-frequency plane containing the most
+            significant tile
+
+        N : `int`
+            estimated number of statistically significant tiles
```

Comment:
What's the utility of this number? Can this be computed externally (i.e. not included here)?

Reply:
@duncanmmacleod, this number is needed to compute the false alarm rate (FAR) expected from white Gaussian noise, which is done in the [**`q_scan` utility**](https://github.com/alurban/gwpy/blob/eventgram/gwpy/signal/qtransform.py#L660). At least one downstream consumer, `gwdetchar-omega`, uses this metric to filter out quiet or insignificant channels, and other consumers (e.g., commissioners writing data quality checks) may also find it useful.

The number of statistically independent tiles is really a property of the search over a `QTiling`, i.e., it's calculated as a sum over all time-frequency planes in the tiling used in a search. So it needs to be made available as part of the `QTiling` object somehow, and this seemed to me like a natural way to do that.